### DISCUSSION TITLE
Having diffrent sources for heating in Energy Dashboard

### DISCUSSION THREAD

**Discussion by caladrien** [home-assistant/frontend#14345] [Dashboard] [answered]
Having gas being tracked by the energy dashboard, I guess, was intended to track your gas consumption. I think may people will use it to track there heating consumption. I’d like to ask either to change the name Gas to Heating or add Heating category to the Dashboard.

In any why the Heating should allow to handle almost all unit. I’m heating with wood pellets. I’m used to ‘to’ or ‘kg’. Oil user would be used to ‘m³’ or ‘L’. When you heat with power it would ‘(k)Wh’.

I’ve managed to convert my kg consumption of pellets, getting form system, to m³. But seeing this in the dashboard with m³ y-axis is quiet meaningless for a pellet user.

![Screenshot 2022-11-09 113852](https://user-images.githubusercontent.com/25111559/200808782-0d69f9c2-d73f-4e30-96b1-5a1f7c1f42f6.png)

**Comment by frenck** [comment, marked as answer]
This is not a frontend-specific request, but a generic Home Assistant request. Please use our community forums for these type of requests: <https://community.home-assistant.io/c/feature-requests/13>

Thanks! 👍

**Reply by caladrien** [reply]
I'll pass it over. Thanks.

**Comment by hzulla** [comment]
@caladrien did you ever get to write a feature request? I had the same thought and didn't find your entry in the community forum, so I wrote a feature request for a heat production diagram here:

https://community.home-assistant.io/t/energy-tab-please-track-generic-heat-production-instead-of-in-addition-to-natural-gas-consumption/785909
